Task-1/Day1__15__a_ProblemSolving.java

```diff
@@ -0,0 +1,37 @@
+import java.util.Scanner;
+
+public class Day1__15__a_ProblemSolving {
+	public static void getPrimeNumbers(int start, int end) {
+		boolean prime[] = new boolean[end + 1];
+
+		// To skip 1 which is neither a prime nor a composite number
+		if (start == 1)
+			start = start + 1;
```

Comment:
The prime number always starts from 2 so no need to check this check. You can assign prime[0], and prime[1] values as false by default;